src/Microsoft.Sbom.Api/Utils/ComponentDetectionCliArgumentBuilder.cs

```diff
@@ -203,5 +203,15 @@ public ComponentDetectionCliArgumentBuilder ParseAndAddArgs(string args)
 
             return this;
         }
+
+        private string AsArgumentValue(string arg)
+        {
+            if (arg.Contains(' '))
+            {
+                return $"\"{arg}\"";
+            }
```

Comment:
This work regardless of the OS, right? Did you test you changes in Mac\Linux\Windows?

Reply:
Yea this is platform independent. But I'll do a sanity check on Ubuntu before merging